backend/internal/handlers/review/service.go

```diff
@@ -0,0 +1,181 @@
+package review
+
+import (
+	"context"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+type ReviewDocument struct {
+	ID        primitive.ObjectID       `bson:"_id,omitempty" json:"_id,omitempty"`
+	Rating    Rating                   `bson:"rating" json:"rating"`
+	Picture   string                   `bson:"picture" json:"picture"`
+	Content   string                   `bson:"content" json:"content"`
+	Reviewer  Reviewer                 `bson:"reviewer" json:"reviewer"`
+	Timestamp time.Time                `bson:"timestamp" json:"timestamp"`
+	Comments  []map[string]interface{} `bson:"comments,omitempty" json:"comments,omitempty"`
+	MenuItem  string                   `bson:"menuItem" json:"menuItem"`
+}
+
+// Rating is a nested struct in ReviewDocument.
+type Rating struct {
+	Portion int  `bson:"portion" json:"portion"`
+	Taste   int  `bson:"taste" json:"taste"`
+	Value   int  `bson:"value" json:"value"`
+	Overall int  `bson:"overall" json:"overall"`
+	Return  bool `bson:"return,omitempty" json:"return,omitempty"`
```

Comment:
Do we anticipate this field being non-existent? I think it should be a required parameter since its part of the review itself and if we're going to have ratings on the other parameters, we're going to additionally have the return value

Reply:
See for changes: [011ad70](https://github.com/GenerateNU/platemate/pull/22/commits/011ad7033bf5a6e7f5a5480180054e963b6d6c29)